src/test/java/uk/gov/ons/census/casesvc/testutil/PubSubHelper.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.ons.census.casesvc.testutil;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.io.IOException;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.BlockingQueue;
+import org.springframework.cloud.gcp.pubsub.core.PubSubTemplate;
+import uk.gov.ons.census.casesvc.model.dto.ResponseManagementEvent;
+
+public class PubSubHelper {
+  private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Please use the `ObjectMapperFactory` once PR https://github.com/ONSdigital/census-rm-case-processor/pull/180 merged